src/contrib/cluster/Akka.Cluster.Sharding/Shard.cs

```diff
@@ -295,7 +295,7 @@ public CurrentShardState(string shardId, IImmutableSet<string> entityIds)
         /// TBD
         /// </summary>
         [Serializable]
-        public sealed class GetShardStats : IShardQuery
+        public sealed class GetShardStats : IShardQuery, IClusterShardingSerializable
```

Comment:
Neither of these messages are persisted, so they couldn't cause the persistence failures included in https://github.com/akkadotnet/akka.net/issues/5116

Reply:
They should be included anyway:
https://github.com/akkadotnet/akka.net/blob/dev/src/contrib/cluster/Akka.Cluster.Sharding/Serialization/ClusterShardingMessageSerializer.cs#L155-L156

Reply:
Yep, that's a good fix but it doesn't close the issue.

Reply:
Do you want me to merge this issue separately?

Reply:
I thought the original header for this issue said "close #5116"

Reply:
no, its only related, found this problem while checking the serializer code